scripts/update-dockerhub-description.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+image_name="all-in-one"
```

Comment:
Note: there are two CI workflows that build Docker images, `ci-docker-build.yml` and `ci-all-in-one-build.yml`. You're only uploading README for one image